portfolio/src/main/webapp/comments.html

```diff
@@ -31,6 +31,16 @@
       <div id="title">
         <h1>Comments</h1>
       </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <button class="btn btn-outline-light" onclick="checkLogStatus()">Log Status</button>
```

Comment:
I think this should say "checkLoginStatus" and "Login Status"!

```suggestion
            <button class="btn btn-outline-light" onclick="checkLoginStatus()">Login Status</button>
```

Reply:
Ack!